flask_celeryext/app.py

```diff
@@ -26,7 +26,11 @@ def setup_task_logger(logger=None, **kwargs):
 def create_celery_app(flask_app):
     """Create a Celery application."""
     celery = current_celery_app
-    celery.conf.update(flask_app.config)
+    if celery_version < '4.0':
```

Comment:
Okay, thinking about this more, this sounds like a good case of premature optimization on my part, will update the PR to use distutils. Thanks!